run-prod.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset -o pipefail
+
+readonly repository="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
+export COMPOSE_PROJECT_NAME="tarpisto-prod"
+source "$repository/scripts/common.sh"
+
+function stop() {
+  pushd "$repository"
+  required_command docker
+  required_command docker-compose
+  docker-compose -f docker-compose.yml -f docker-compose.prod.yml down -v || true
+  popd
+}
+trap stop EXIT
+
+function main() {
+    required_command npm
+    required_command docker
+    required_command docker-compose
+
+    pushd "$repository"
+
+    export PORT=${PORT:-"9000"} # Docker-compose exposes this port in 9020
+
+    export USER_SERVICE_SERVICE_ID=${USER_SERVICE_SERVICE_ID:-"11188b9c-9534-4faf-8355-60973b720647"}
+    export USER_SERVICE_URL=${USER_SERVICE_URL:-"http://127.0.0.1:8080"}
+    export USER_SERVICE_SECRET=${USER_SERVICE_SECRET:-"catlike-meringue-tying-PASTERN-bed-simply"}
```

Comment:
We should **absolutely** fail and scream if we don't have a secret set **in production**!! This is an `exit 1` type of situation.

Reply:
`"There are only two hard things in Computer Science: cache invalidation and naming things." --Phil Karlton`

`run-prod.sh` is meant to be only for running prod-like environment **locally** and therefore env values has default values for easier testing. Not in real production environment. Use case for this is to be able check Docker image is valid and working as dev server does not run inside Docker. There are cases where dev server and built prod version of Next.js does things a little differently and this is meant only for local testing purposes.

This was mentioned in `README.md`, but clearly this needs more clarification to name: TODO. I did not make big difference in naming local scripts as previous system used only Docker image for real production environment. Same goes for docker-compose files, them are named in local use only manner as previous version of Tärpistö.

If running Dockerfile itself, it will only use `start-prod-server.sh` inside the image and that script has purposely no default values for these. Starting Next.js will check that all needed variables should be initialized.

Reply:
Renamed the script to scream about being local one. Also reworked env values fetching and scripts now will tell in which kind of env them are run on. Now it's more clear that prod gets only `NODE_ENV`, `APP_ENV` and `PORT` by default with prod-safe values.

Reply:
There will be need to have env variables at the build time set as Next's build system emulates running server and our env variables missing check kicks in, but when starting it with no variables set after Docker image build, it will scream missing env variables.